### PR TITLE
Refactor for NfN organization

### PR DESCRIPTION
**Pull request by mcbouslog**
Refactor Field Book to be based on single project to organization of projects.

https://field-book-preview.notesfromnature.org/

### CODE REVIEW

File: src/containers/BadgeContainer.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ class BadgeContainer extends React.Component {
   constructor() {
     super();
     this.state = {
-      caesarData: [],
+      caesarData: new Map(),
```
Comment (eatyourgreens) [inline src/containers/BadgeContainer.jsx]: Be careful with this in IE11, which doesn't support `map.values()`.
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Browser_compatibility

File: src/context/StatsContext.jsx [hunk 2]
```diff
@@ -2,111 +2,132 @@ import PropTypes from 'prop-types';
...
-        });
-    } else {
-      this.setState({
-        collectiveStatsByDay: [],
-        userStatsByDay: [],
-        userStatsByMonth: []
+    const { explorer, projects } = this.props;
+    if (explorer && explorer.id && projects && projects.length) {
+      projects.forEach(project => {
+        statsClient
```
Comment (eatyourgreens) [inline src/context/StatsContext.jsx]: I could be wrong, but I think the old stats service is being retired in favour of the new GraphQL service, so the queries here might change in future.

File: src/containers/BadgeContainer.jsx (hunk 1 above)
Comment (mcbouslog) [reply src/containers/BadgeContainer.jsx]: Thank you for the heads up! I'll check with the project lead, keep an eye out for, and see if I can discern from Google Analytics if it'll be an issue.

File: src/context/StatsContext.jsx (hunk 2 above)
Comment (mcbouslog) [reply src/context/StatsContext.jsx]: Towards top of to dos is refactor for new GraphQL Stats service for few reasons.

**Comment by mcbouslog** [comment]
Closing for further review of Caesar and stats API requests.

**Pull request closed without merging**